service/history/queues/executable.go

```diff
@@ -170,6 +174,9 @@ func NewExecutable(
 			return false
 		},
 		DLQWriter: nil,
+		AttemptsBeforeSendingToDlq: func() int {
+			return 0
```

Comment:
should be MaxInt by default, meaning never to DLQ?

Reply:
Yep. That is a good idea.